.circleci/config.yml

```diff
@@ -798,13 +809,15 @@ workflows:
                 - dev
                 - stage
                 - prod
-      - deploy-reporter-tools:
```

Comment:
I'm debating whether we should put this back (and allow deployment from the `prod` branch) to ease the transition to having the reporter in dev/stage/prod.